modules/@apostrophecms/template/index.js

```diff
@@ -643,6 +643,7 @@ module.exports = {
           modules: {},
           prefix: req.prefix,
           sitePrefix: self.apos.prefix,
+          shortName: self.apos.shortName,
```

Comment:
_ℹ️  pass the `shortName` to the frontend so it's used to retrieve the "logged-in flag" cookie_

Reply:
tests added in f9566b2ff